source/module_md/test/FIRE_test.cpp

```diff
@@ -0,0 +1,116 @@
+#include "setcell.cpp"
+#include "gtest/gtest.h"
+#include "../FIRE.h"
+
+UnitCell_pseudo ucell;
+class FIRE_test : public testing::Test
+{
+protected:
+	Verlet *verlet;
+
+	static void SetUpTestCase()
+	{
+		setupcell(ucell);
+        parameters();
+	}
+
+	static void TearDownTestCase(){}
+};
+
+TEST_F(FIRE_test, setup)
+{
+    verlet = new FIRE(INPUT.mdp, ucell);
```

Comment:
Setup and Tear down should be placed in test class.

Reply:
OK, I will update it.